ye.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///animals.db'
+db = SQLAlchemy(app)
+
+class Animal(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    breed = db.Column(db.String(100), nullable=False)
+    photo = db.Column(db.String(200), nullable=False)
+    date_of_birth = db.Column(db.DateTime, nullable=False)
+
+    @property
+    def age(self):

```

Comment:
Better to add computed field in pydantic model https://docs.pydantic.dev/2.0/usage/computed_fields/